### ISSUE TITLE
Add scss-lint configuration

### ISSUE THREAD

**Issue by stelioschar** [MozaikAgency/wp-theme-bootstrap#7]
Propose a scss-lint configuration file. 

Until now the most stable scss linter seems to be the [scss-lint](https://github.com/brigade/scss-lint).

Unfortunately this plugin is strongly dependent to ruby, so it would be tricky to implement this to the existing gulp workflow.

On the other hand you can have a global scss-lint configuration file and use it through your IDE(e.g. PHPStorm).

**Comment by stelioschar** [comment]
a default configuration file added in issue #8

**Comment by Maximilianos** [comment]
This project: sass-lint, https://github.com/sasstools/sass-lint
is free of the ruby dependency. What do you think of it?

**Comment by Maximilianos** [comment]
Actually just saw this issue https://github.com/sasstools/sass-lint/issues/6
looks like they are not ready for the prime time quite yet. They still do not have all the features of ruby scss-lint

**Comment by Maximilianos** [comment]
a shame.. would be nice to include in the project

**Comment by stelioschar** [comment]
Exactly,

But we could give it a try, just to be ready for the next releases of [the sass-lint](https://github.com/sasstools/sass-lint)

**Comment by Maximilianos** [comment]
yep I will create a branch where sass-lint is configured in the project and when sass-lint is launched we can merge.

**Comment by Maximilianos** [comment]
Maybe consider https://github.com/stylelint/stylelint
It supports scss I think

**Comment by ntwb** [comment]
> Maybe consider https://github.com/stylelint/stylelint

WordPress CSS Coding Standards shared config https://github.com/ntwb/stylelint-config-wordpress

> It supports scss I think

It does 😄

**Comment by Maximilianos** [comment]
Thanks for the suggestion @ntwb will check it out :)
